dataservice/api/common/validation.py

```diff
@@ -36,3 +37,41 @@ def validate_kf_id(prefix, value):
     m = re.search(r, value)
     if not m:
         raise ValidationError('Invalid kf_id')
+
+
+class EnumValidator(OneOf):
```

Comment:
This class isn't doing anything differently than `OneOf` anymore, since you removed the part that made the validation case insensitive. I think we can probably get rid of this class and then wherever you have `EnumValidator` you can just replace that with `OneOf` (which I think is just in `enum_validation_generator`)